schemas/6.json

```diff
@@ -0,0 +1,1121 @@
+        "fields": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Field"
+          }
+        }
+      },
+      "required": [
+        "fields"
+      ],
+      "title": "Form"
+    },
+    "Field": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "variable": {
+          "type": "string"
+        },
+        "prompt": {
+          "type": "string"
+        }
+      },
+      "required": [
+        "variable",
+        "prompt"
+      ],
+      "title": "Field"
+    },
+    "Stage": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "interviewScript": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Narrative",
+            "AlterForm",
+            "AlterEdgeForm",
+            "EgoForm",
+            "NameGenerator",
+            "NameGeneratorQuickAdd",
+            "NameGeneratorList",
+            "NameGeneratorAutoComplete",
+            "Sociogram",
+            "DyadCensus",
+            "TieStrengthCensus",
+            "Information",
+            "OrdinalBin",
+            "CategoricalBin"
+          ]
+        },
+        "label": {
+          "type": "string"
+        },
+        "form": {
+          "$ref": "#/definitions/Form"
+        },
+        "quickAdd": {
+          "type": ["string", "null"]
+        },
+        "createEdge": {
+          "type": "string"
+        },
+        "dataSource": {
+          "type": ["string", "null"]
+        },
+        "subject": {
+          "$ref": "#/definitions/Subject"
+        },
+        "panels": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Panel"
+          }
+        },
+        "prompts": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Prompt"
+          },
+          "minItems": 1
+        },
+        "presets": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Preset"
+          },
+          "minItems": 1
+        },
+        "background": {
+          "type": "object",
+          "items": {
+            "$ref": "#/definitions/Background"
+          },
+          "minItems": 1
+        },
+        "sortOptions": {
+          "type": "object",
+          "items": {
+            "$ref": "#/definitions/SortOptions"
+          }
+        },
+        "cardOptions": {
+          "type": "object",
+          "items": {
+            "$ref": "#/definitions/CardOptions"
+          }
+        },
+        "searchOptions": {
+          "type": "object",
+          "items": {
+            "$ref": "#/definitions/SearchOptions"
+          }
+        },
+        "behaviours": {
+          "type": "object",
+          "items": {
+            "$ref": "#/definitions/Behaviours"
+          },
+          "minItems": 1
```

Comment:
Other similar object keys don't have a minItems. I'm not really sure if they should be updated, or if this one shouldn't have any minimum. Thoughts?

Reply:
I thought that was odd too. This value hasn't been changed from schema 5. Perhaps we should open an issue to remove it where it isn't needed?